Realm/Realm/Transaction.cs

```diff
@@ -23,6 +23,28 @@
 
 namespace Realms
 {
+        /// <summary>
+        /// Represents the state of a <see cref="Transaction"/>.
+        /// </summary>
+    [Flags]
```

Comment:
This should probably not be flags - MS has pretty good docs about what `[Flags]` designates and when to use it [here](https://learn.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=net-7.0).